src/app/learn/learn.component.ts

```diff
@@ -6,7 +6,10 @@ import { Component } from '@angular/core';
 })
 export class LearnComponent {
   public sidebarRoutes: any[] = [
-    { name: 'Overview', path: '/learn' },
+    { name: 'Overview', path: '/learn', children: [
+      { name: 'Test1', path: '/learn/test-1' },
+      { name: 'Test2', path: '/learn/test-2' }
+    ]},
```

Comment:
But... Can we count on the routes being passed in this way?   Aren't these routes coming from Angular2 / skyUX builder based on file structure?